site/packages/aa-alchemy/signer/authenticate.md

```diff
@@ -0,0 +1,69 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: Alchemy Signer • authenticate
+  - - meta
+    - name: description
+      content: Learn how to use the AlchemySigner.authenticate method
+  - - meta
+    - property: og:description
+      content: Learn how to use the AlchemySigner.authenticate method
+  - - meta
+    - name: twitter:title
+      content: Alchemy Signer • authenticate
+  - - meta
+    - name: twitter:description
+      content: Learn how to use the AlchemySigner.authenticate method
+---
+
+# authenticate
+
+The `authenticate` method is used to authenticate a user with the Alchemy Signer.
+
+## Usage
+
+::: code-group
+
+```ts
+import { signer } from "./signer";
+
+const bundlePromise = new Promise(async (resolve) => {
+  const otpFromUser = await getOtpFromUser();
+  resolve(otpFromUser);
+});
+
+const user = await signer.authenticate({
+  type: "email",
+  email: "user@email.com",
+  bundle: bundlePromise,
+});
+```
+
+<<< @/snippets/signers/alchemy/signer.ts
+
+:::
+
+## Returns
+
+`Promise<User>` -- on success returns a `User` object representing the authenticated user.
+
+## Parameters
+
+`AuthParams` -- an object that contains the following properties:
+
+```ts
+export type AuthParams =
+  | { type: "email"; email: string; bundle: Promise<string> }
```

Comment:
we should put a clear explanation of what bundle is here and what it is doing/purpose.